### PR TITLE
Logcat printer

### PR DESCRIPTION
**Pull request by Fatme**
Should be merged after this https://github.com/telerik/mobile-cli-lib/pull/304

### CODE REVIEW

**Comment by fealebenpae** [comment]
ping @dtopuzov

**Comment by fealebenpae** [comment]
:+1:

File: lib/providers/logcat-printer.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import Future = require("fibers/future");
```
Comment (teobugslayer) [inline lib/providers/logcat-printer.ts]: Is this needed?

**Comment by teobugslayer** [comment]
Note, that the new --just option is not documented in our html help. Please add relevant help where needed.

**Comment by Fatme** [comment]
I've updated documentation.

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**